packages/blade/rollup.config.js

```diff
@@ -123,9 +124,35 @@ const getDeclarationsConfig = ({ exportCategory }) => ({
   ],
 });
 
+const getCSSVariablesConfig = ({ exportCategory }) => ({
+  input: `src/${exportCategory}/index.ts`,
+  output: {
+    file: `${outputRootDirectory}/js-bundle-for-css/${exportCategory}Bundle.js`,
+    format: 'cjs',
+  },
+  plugins: [
+    pluginPeerDepsExternal(),
+    pluginResolve({ extensions: webExtensions }),
+    pluginCommonjs(),
+    pluginBabel({
+      exclude: 'node_modules/**',
+      babelHelpers: 'runtime',
+      envName: 'production',
+      extensions: webExtensions,
+    }),
+    aliases,
+  ],
+});
+
 const config = () => {
   const framework = process.env.FRAMEWORK;
+  const generateCSSVariables = process.env.GENERATE_CSS_VARIABLES;
 
+  if (generateCSSVariables == 'true' && framework === 'REACT') {
```

Comment:
nit: @chaitanyadeorukhkar Any specific reason to not do a strict check here for `generateCSSVariables`?

Reply:
No reason. This is derived from env variables which in general are always transformed into a string.